mlir/include/mlir-c/Dialect/SparseTensor.h

```diff
@@ -52,16 +54,15 @@ mlirAttributeIsASparseTensorEncodingAttr(MlirAttribute attr);
 
 /// Creates a `sparse_tensor.encoding` attribute with the given parameters.
 MLIR_CAPI_EXPORTED MlirAttribute mlirSparseTensorEncodingAttrGet(
-    MlirContext ctx, intptr_t lvlRank,
-    enum MlirSparseTensorLevelType const *lvlTypes, MlirAffineMap dimToLvl,
-    MlirAffineMap lvlTodim, int posWidth, int crdWidth);
+    MlirContext ctx, intptr_t lvlRank, MlirBaseLevelType const *lvlTypes,
```

Comment:
e.g. here we want to use MlirSparseTensorLevelType (and not a base)

Reply:
you mean the typedef after I switch the name right?